tests/foreman/cli/test_organization.py

```diff
@@ -1693,3 +1694,32 @@ def test_positive_info_by_name(self):
         org = make_org()
         result = Org.info({'name': org['name']})
         self.assertEqual(org['id'], result['id'])
+
+    @tier1
+    @skip_if_bug_open('bugzilla', 1418412)
+    def test_positive_multibyte_latin1_org_names(self):
+        """Hammer Multibyte and Latin-1 Org names break list pagination
+
+        :id: 4fa0afe7-6d0a-4c3e-a0fc-4ecb95c50fc9
+
+        :BZ: 1418412
+
+        :expectedresults: Multibyte and latin1 names need to be
+            displayed with consistent spacing
+        """
+        org_names = valid_org_names_list()
+        for org in org_names:
+            make_org({'name': org})
+        org_list = Org.list(output_format='table').remove('')
```

Comment:
In addition, `remove()` will delete only the first found element (empty string in your case), so in case there're few empty lines, this approach won't work. Try
```python
org_list = [line for line in Org.list(output_format='table') if line]
```
instead.